Documentation/guides/profiling.md

```diff
@@ -214,6 +225,8 @@ Mono:
     Methods JITted using LLVM           : 0
     Methods using the interpreter       : 0
 
+_NOTE: that `counters.txt` is not available in .NET 6 projects._
```

Comment:
The code that generates `counters.txt` is still active in NET6 and `dotnet/mono` runtime appears to still [support](https://github.com/dotnet/runtime/blob/a0a5fc9771f63aa31cb45b3577a7cea31527a2fb/src/mono/mono/mini/mini-runtime.c#L4949-L4994) [`mono_counters_dump`](https://github.com/dotnet/runtime/blob/a0a5fc9771f63aa31cb45b3577a7cea31527a2fb/src/mono/mono/utils/mono-counters.c#L619-L662) API we use (and it's not marked as obsolete), it should still work?

Reply:
I tried a Debug build and `-c Release -p:RunAOTCompilation=false` (to turn off AOT).

This appears to work fine in Xamarin.Android projects in either of those configurations, but not .NET 6?

Reply:
Hm, can it be due to missing FS write permissions for the test app?

Reply:
No the file is there, it's just mostly blank like:
```
## Runtime.register: type=Microsoft.NetConf2021.Maui.MainApplication, Microsoft.NetConf2021.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
## Runtime.register: type=Microsoft.Maui.MauiApplication, Microsoft.Maui, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
## Runtime.register: type=Microsoft.Maui.MauiApplication+ActivityLifecycleCallbacks, Microsoft.Maui
```
Each line has several empty lines in between.

Reply:
So it appears the counters are disabled in dotnet/mono, even though code is there.